binding/web/src/types.ts

```diff
@@ -10,7 +10,7 @@
 */
 
 export type LeopardInitConfig = {
-  /** @defaultValue true */
+  /** @defaultValue false */
```

Comment:
why are there two config classes here instead of a single type holding all the optionals? Also, I believe we talked about naming the options/config types `${Engine}Options`. If we go with config that's fine, but we need to ensure everyone knows to name their type that.